sdk/python/tests/components/test_python_op.py

```diff
@@ -228,7 +229,10 @@ def calc_pipeline(
             #Compiling the pipleine:
             pipeline_filename = str(Path(temp_dir_name).joinpath(calc_pipeline.__name__ + '.pipeline.tar.gz'))
             import kfp.compiler as compiler
-            compiler.Compiler().compile(calc_pipeline, pipeline_filename)
+
+            with warnings.catch_warnings():  # ignore pending deprecation warnings
```

Comment:
Which deprecated member does this use?

Reply:
If `artifact_location` is not provided, I generates a pending deprecation warning and return the default value for now (e.g. minio-service:9000, ...).

Reply:
removed after sync with latest HEAD where default artifact location is empty.